package.json

```diff
@@ -82,7 +82,7 @@
     "@types/tmp": "^0.1.0",
     "@types/uuid": "^3.4.4",
     "codecov": "^3.0.0",
-    "eslint": "^5.0.0",
+    "eslint": "^5.16.0",
```

Comment:
I think we should avoid introducing dep updates to the repo itself if the scope of this PR is just fine tuning samples.

Reply:
Agree. I changed it back.